src/cdk/testing/testbed/fake-events/event-objects.ts

```diff
@@ -41,11 +41,11 @@ export function createMouseEvent(type: string, clientX = 0, clientY = 0, button
 
   // `initMouseEvent` doesn't allow us to pass the `buttons` and
   // defaults it to 0 which looks like a fake event.
-  Object.defineProperty(event, 'buttons', {get: () => 1});
+  Object.defineProperty(event, 'buttons', {get: () => 1, configurable: true});
```

Comment:
What do you think about creating a helper function that adds the `configurable: true` so we don't need to repeat it throughout the file? That would also give us a single place on which we can comment on why it's there.

Reply:
I like it. Done! I called `defineReadonlyEventProperty` as that matches exactly with how the default properties are described for DOM events.